core/src/wallet/bitcoin/api_impl/BitcoinLikeTransactionApi.h

```diff
@@ -144,6 +146,9 @@ namespace ledger {
                                                    const api::Currency &currency,
                                                    const std::string &keychainEngine);
 
+            static int64_t computeDustAmount(const api::Currency &currency, int32_t size);
```

Comment:
What is the rational to have a signed `size` here ? I think negative size doesn't make any sense both for the computation and the blockchain protocol.

Reply:
I agree with you. But the problem here is that we are forced to use signed integers to manipulate sizes (for example EstimatedSize object) because we are exposing those objects and functions via djinni (and unfortunately djinni doesn't support unsigned integers to keep compliance with java)